src/header/Header.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { StatusBar, StyleSheet, Text, View } from 'react-native';
+import isEmpty from 'lodash.isempty';
+import Icon from '../icons/Icon';
+
+export const DummyNavButton = () => (
```

Comment:
can we rename to `DefaultNavButton` if that makes sense in this context? 
I'll leave it up to you to the decide.